src/main/resources/liquibase/202309271140-create-tally-state-table.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+  <changeSet id="202309271140-1" author="mstead">
+    <comment>Create tally_state table</comment>
+
+    <createTable tableName="tally_state">
+      <column name="org_id" type="VARCHAR(32)"/>
+      <column name="service_type" type="VARCHAR(32)"/>
+      <column name="latest_event_record_date" type="TIMESTAMP WITH TIME ZONE"/>
+    </createTable>
+
+    <addPrimaryKey constraintName="tally_state_pkey"
+      tableName="tally_state"
+      columnNames="org_id, service_type"/>
+
+  </changeSet>
```

Comment:
This change set is missing the rollback statement.

Reply:
DONE